### PR TITLE
Fix a colour issue and cleaned up the code.

### PR DESCRIPTION
**Pull request by psherwinnicholls**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/DecodedCo/brand-guidelines/pull/55" target="_blank">ReviewNinja

File: pages/how-to/signature.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ Emails are our primary form of communication with clients. This tool will allow
...
+    <a id="worknum" contenteditable="true" class="worknum" style="box-shadow:0 0 0 0 !important; font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0; padding-bottom:1px;" href="tel:+442035830972">+44 20 3583 0972</a><span style="color:#BAB8B0;" contenteditable="true">&nbsp;&nbsp;|&nbsp;&nbsp;</span><a class="directnum" id="directnum" contenteditable="true" class="number" style="box-shadow:0 0 0 0 !important;  font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0; padding-bottom:1px;" href="tel:+441234567891">+44 YOUR NUMBER</a>
```
Comment (jonashaefele) [inline pages/how-to/signature.md]: when you have long lines like these split them on several lines so git diffs become readable... eg

```

<a 
  id="worknum" 
  contenteditable="true" 
  class="worknum" 
  style="box-shadow:0 0 0 0 !important; 
        font-size:14px; 
        font-weight:200 !important; 
        color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; 
        text-decoration:none;
        border-bottom:1px solid #BAB8B0; 
        padding-bottom:1px;" 
  href="tel:+442035830972">
    +44 20 3583 0972
</a>
<span style="color:#BAB8B0;" contenteditable="true">   
  &nbsp;&nbsp;|&nbsp;&nbsp;
</span>
...
```

Comment (psherwinnicholls) [inline pages/how-to/signature.md]: I thought that it was standard to 'minify' the styles for html emails? happy to change though :)

**Comment by jonashaefele** [comment]
lovely! :+1:

**Pull request merged**